recipe/bld.bat

```diff
@@ -6,6 +6,16 @@
 @rem that it doesn't block the terminal, but we also can't get the return
 @rem value for the conda build tests.
 
+curl -C - -o "mingw-w64-x86_64-pcre2-10.34-1-any.pkg.tar.xz" -SLO http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-pcre2-10.34-1-any.pkg.tar.xz
```

Comment:
Aye! That is cool!

Reply:
That shouldn't be needed anymore. There is now a conda package for that.